index.md

```diff
@@ -41,8 +41,9 @@ can and should be stored in a version control system.
 
 > ## Prerequisites {.prereq}
 >
-> A short paragraph describing what learners need to know
-> before tackling this lesson.
+> In this lesson we use Git from the Unix Shell
```

Comment:
I'd rephrase this sentence as: "In this lesson we use Git from the Unix Shell. Some previous experience with the shell is expected, *but isn't mandatory*." I think that's a little shorter and clearer.